src/utils/aepp-wallet-communication/rpc/aepp-rpc.js

```diff
@@ -46,6 +47,10 @@ const RESPONSES = {
       if (msg.result) instance.rpcClient.info.status = RPC_STATUS.CONNECTED
       instance.rpcClient.processResponse(msg)
     },
+  [METHODS.bridge]: (instance) =>
+    (msg) => {
+      instance.rpcClient.processResponse(msg)
```

Comment:
I think reasonable to combine it with `METHODS.updateNetwork` notification to receive a new URL at the same time as networkId/network changed. In general network change on the wallet side is not covered as I see.

Reply:
Oh yeah. Got it, Let me modify the `METHODS.updateNetwork` notification response to include a new URL.